src/lib/getVars.ts

```diff
@@ -5,12 +5,19 @@ import * as core from '@actions/core'
 const { GITHUB_REPOSITORY, RUNNER_TOOL_CACHE } = process.env
 const CWD = process.cwd()
 
+export enum Strategy {
+  CopyImmutable = 'copy-immutable',
+  Copy = 'copy',
+  Move = 'move',
+}
+
```

Comment:
Prefer union types, because TS enums are kinda bad but mostly because of our stylistic choices at Masterworks.

```suggestion
export const STRATEGIES = ['copy-immutable', 'copy', 'move'] as const
export type Strategy = typeof STRATEGIES[number]
```